modules/exampleauth/src/Auth/Source/UserClick.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimpleSAML\Module\exampleauth\Auth\Source;
+
+use Exception;
+use SimpleSAML\Auth;
+use SimpleSAML\Assert\Assert;
+use SimpleSAML\Error;
+use SimpleSAML\Logger;
+use SimpleSAML\Module;
+use SimpleSAML\Utils;
+use Symfony\Component\HttpFoundation\{Request, Response};
+
+use function array_key_exists;
+use function sprintf;
+
+/**
+ * Profileauth authentication source.
+ *
+ * This class is an authentication source which stores all users in an array,
+ * and authenticates users by clicking on their name.
+ *
+ * @package SimpleSAMLphp
+ */
+
+class UserClick extends Auth\Source
```

Comment:
I'm confused as to the difference between `Profileauth` (l 20) and `UserClick`. What do we call this auth source?